ext/intl/dateformat/dateformat.c

```diff
@@ -44,6 +44,10 @@ void dateformat_register_constants( INIT_FUNC_ARGS )
 	DATEFORMATTER_EXPOSE_CLASS_CONST( MEDIUM );
 	DATEFORMATTER_EXPOSE_CLASS_CONST( SHORT );
 	DATEFORMATTER_EXPOSE_CLASS_CONST( NONE );
+	DATEFORMATTER_EXPOSE_CUSTOM_CLASS_CONST( "RELATIVE_FULL", UDAT_FULL_RELATIVE );
+	DATEFORMATTER_EXPOSE_CUSTOM_CLASS_CONST( "RELATIVE_LONG", UDAT_LONG_RELATIVE );
+	DATEFORMATTER_EXPOSE_CUSTOM_CLASS_CONST( "RELATIVE_MEDIUM", UDAT_MEDIUM_RELATIVE );
+	DATEFORMATTER_EXPOSE_CUSTOM_CLASS_CONST( "RELATIVE_SHORT", UDAT_SHORT_RELATIVE );
```

Comment:
Any particular motivation to invert the order relative to the ICU names?

Reply:
When looking up these constants on Google, I mainly found them this way (e.g. https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/DateFormat.html#RELATIVE_FULL, or https://docs.microsoft.com/en-us/dotnet/api/android.icu.text.dateformat.relativefull?view=xamarin-android-sdk-9). So I thought it's probably better to change the order...

Reply:
Makes sense.